site/assets/js/color-modes/index.js

```diff
@@ -0,0 +1,62 @@
+/*!
```

Comment:
@XhmikosR and/or @GeoSot Would be great if you could look at it the JS part of this PR 🙏 You'll have better feedback to do than me.